packages/node/test/integration/cross-node-multisig-creation.spec.ts

```diff
@@ -65,14 +76,36 @@ describe("Node can create multisig, other owners get notified", () => {
     firebaseServiceFactory.closeServiceConnections();
   });
 
-  it("Node A can create multisig and sync with Node B on new multisig creation", async () => {
-    const multisigAddress = await getNewMultisig(nodeA, [
+  it("Node A can create multisig and sync with Node B on new multisig creation", async done => {
+    const ownersPublicIdentifiers = [
       nodeA.publicIdentifier,
       nodeB.publicIdentifier
-    ]);
-    const openChannelsNodeA = await getChannelAddresses(nodeA);
-    const openChannelsNodeB = await getChannelAddresses(nodeB);
-    expect(openChannelsNodeA[0]).toEqual(multisigAddress);
-    expect(openChannelsNodeB[0]).toEqual(multisigAddress);
+    ];
+    nodeA.on(
+      NODE_EVENTS.CREATE_CHANNEL,
+      async (data: NodeTypes.CreateChannelResult) => {
+        await confirmChannelCreation(
+          nodeA,
+          nodeB,
+          ownersPublicIdentifiers,
+          data
+        );
+        done();
+      }
+    );
+    await getMultisigCreationTransactionHash(nodeA, ownersPublicIdentifiers);
   });
 });
+
+async function confirmChannelCreation(
+  nodeA: Node,
+  nodeB: Node,
+  ownersPublicIdentifiers: string[],
+  data: NodeTypes.CreateChannelResult
+) {
+  const openChannelsNodeA = await getChannelAddresses(nodeA);
+  const openChannelsNodeB = await getChannelAddresses(nodeB);
+  expect(openChannelsNodeA[0]).toEqual(data.multisigAddress);
+  expect(openChannelsNodeB[0]).toEqual(data.multisigAddress);
```

Comment:
Replace `toEqual` with `toBe` for string comparisons. `toEqual` is intended for object equality